modules/ROOT/pages/composite-databases/administration.adoc

```diff
@@ -83,12 +83,12 @@ The link:{neo4j-docs-base-uri}/cypher-manual/{page-version}/aliases#alias-manage
 SHOW ALIASES FOR DATABASE
```

Comment:
Not related to my update but the sentence on line 76 feels more like it belongs with this example and should be under the same header and not before the header?

Reply:
Thanks, @Hunterness - we will soon reorganise this page (as part of the migration from Cypher Manual), and will take this into consideration when we get to it  (unless @renetapopova wants to take action before).